config/samples/envoy_v1alpha1_envoybootstrap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: envoy.marin3r.3scale.net/v1alpha1
+kind: EnvoyBootstrap
+metadata:
+  name: envoybootstrap-sample
+spec:
+  discoveryService: instance
+  clientCertificate:
+    directory: /etc/envoy/tls/client
+    secretName: discoveryservice-client-certificate
+    duration: 1h
+  envoyStaticConfig:
+    configMapNameV2: envoy-bootstrap-v2
```

Comment:
Have you thought about maybe, instead of having fields with specific envoy version, which requires new fields for every envoy version, having 2 immutable fields that can work with any envy version (`currentVersion` and `previousVersion`, or something like that)?

Reply:
As discussed offline, being this an internal resource I'd rather leave it explicit like this. Also, being internal it's easier to change if we see it doesn't fit.